components/modal.css

```diff
@@ -0,0 +1,70 @@
+@import url(../styles.css);
+
+.modals-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 12rem;
+    max-height: auto;
+    box-shadow: 1px 1px 10px 8px rgba(0,0,0,0.1);
+    margin: 0 4rem;
+}
+
+.modal-content{
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    justify-content: center;
+    align-items: center;
+    background-color: rgb(0,0,0);
+    background-color: rgba(0,0,0,0.4);
+}
+
+.modal{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    top: 40%;
+    left: 40%;
```

Comment:
can be in rem

Reply:
if I use rem here then the modal has given a fixed margin and it does not come in a center if we adjust the screen. that's why I'm not using rem here. hope you understand.